bats/fb-test-foreman.bats

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bats
+# vim: ft=sh:sw=2:et
+
+set -o pipefail
+
+@test "check web app is up" {
+  curl -sk "https://localhost$URL_PREFIX/users/login" | grep -q login-form
+}
+
+@test "wake up puppet agent" {
+  source ~/.bashrc
+  puppet agent -t -v
+}
+
+@test "check web app is still up" {
+  curl -sk "https://localhost$URL_PREFIX/users/login" | grep -q login-form
+}
+
+@test "check smart proxy is registered" {
+  hammer --csv proxy list | grep -q "$(hostname -f)"
+}
+
+@test "check host is registered" {
+  hammer host info --name "$(hostname -f)"
+}
+
+@test "check hammer ping" {
+  hammer ping
+}
```

Comment:
This file does not appear to differ that much from `fb-test-katello.bats`. Could we centralize those into a `fb-health-check.bats` or something to that effect?